docs/changes.rst

```diff
@@ -162,6 +162,9 @@ Released: not yet
   instances for Open/Pull after  the request to the server is complete). (see
   issue #1801)
 
+* Added security issues 50463  50792  50748  51159 50885 50886  to Makefile
+  ignore list of new security issue August and September 2022.
```

Comment:
This change adds less safety issues than what this change entry says.

Reply:
Fixed